src/XRP/core-xrpl-client.ts

```diff
@@ -422,7 +443,10 @@ export default class CoreXrplClient implements CoreXrplClientInterface {
       rawTransactionStatus = await this.getRawTransactionStatus(transactionHash)
     }
     /* eslint-enable no-await-in-loop */
-
-    return rawTransactionStatus
+    const lastLedgerPassed = latestLedgerSequence >= lastLedgerSequence
+    return {
+      rawTransactionStatus: rawTransactionStatus,
```

Comment:
is it possible for `rawTransactionStatus` to NOT start with `tes` and have `lastLedgerPassed === true`? 

my gut tells me that the value of `lastLedgerPassed` is dividing two disjoint sets in `rawTransactionStatus`

Reply:
put another way - i suspect that the trickiness here revolves around the fact that this function can return `TransactionStatus.Pending` at all, which is not a `FinalTransactionStatus`

Reply:
got it the above comment doesn't make sense because `rawTransactionStatus` is a diff object. dang